### PR TITLE
Update google_maps.markdown

### PR DESCRIPTION
**Pull request by inverse**
Provide additional information to track devices through this integration.

### CODE REVIEW

File: source/_integrations/google_maps.markdown [hunk 1]
```diff
@@ -28,6 +28,10 @@ device_tracker:
     username: YOUR_USERNAME
 ```
 
+Once enabled and you have rebooted devices discovered through this integration wil be listed in the `known_devices.yaml`file within your configuration directory.
```
Comment (klaasnicolaas) [inline source/_integrations/google_maps.markdown:31]: Did we not already say goodbye to the `known_devices.yaml` file?

Comment (inverse) [reply source/_integrations/google_maps.markdown:31]: I'm not aware of such thing. That being said I'm not that actively involved in the core.

Comment (frenck) [reply source/_integrations/google_maps.markdown:31]: It is still used.

**Review by frenck — APPROVED** [review]
Thanks, @inverse! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**